include/slam_toolbox/visualization_utils.hpp

```diff
@@ -99,6 +109,14 @@ inline visualization_msgs::msg::InteractiveMarker toInteractiveMarker(
 
   return int_marker;
 }
+template<class NodeT>
+inline visualization_msgs::msg::InteractiveMarker toInteractiveMarker(
+  visualization_msgs::msg::Marker & marker,
+  const double & scale,
+  NodeT && node)
+{
+  return toInteractiveMarker(marker, scale, clock);
```

Comment:
`clock` isn't valid in this function scope, does this compile?

Reply:
Oh sorry, I didn't notice because could compile it. thanks